zenoh_app/pose_service.py

```diff
@@ -104,8 +101,15 @@ def setGoal(self, lat, lon):
 
     def engage(self):
         self.publisher_gate_mode.put(GateMode(data=GateMode.DATA['AUTO'].value).serialize())
+        time.sleep(1)
```

Comment:
Same here. Add some comments for why we need sleep. This will help others understand the code.

Reply:
As mentioned in comment above, the redundant sleep functions have been removed.